following_angle_deg         : 0.0       # theta_sh [deg] 
following_distance          : 0.7       # l_0 [m] 
spring_constant_linear      : 30.0      # k_1 [N/m] 
spring_constant_angular     : 0.01      # k_2 [N・m/rad] 
weight_robot                : 30.0      # M [Kg] 
moment_inertia              : 15.0      # I [Kg・m^2] 
viscous_friction_linear     : 1.00      # k_3 [N・s/m] 
viscous_friction_angular    : 1.00      # k_4 [N・s/rad] 
radius_robot                : 0.3       # L [m] 